liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -1353,7 +1364,16 @@ private void runCompileMojo(String goal) throws MojoExecutionException {
      * @throws MojoExecutionException
      */
     private void runCompileMojoLogWarning() throws MojoExecutionException {
-        runCompileMojo("compile");
+        runCompileMojo("compile", null);
```

Comment:
Any reason to pass in `null` vs just passing in `project`?

Reply:
Thanks, I've simplified the logic